exetera/core/dataframe.py

```diff
@@ -435,6 +435,40 @@ def apply_index(self, index_to_apply, ddf=None):
                 field.apply_index(index_to_apply, in_place=True)
             return self
 
+    def sort_values(self, by, ascending=True, ddf=None):
+        """
+        Sort by the values
+        
+        :param by: Name (str) or list of names (str) to sort by.
+        :param ascending: bool or list of bool, default True. Sort ascending vs. descending. 
+        :param ddf: optional - the destination data frame
+        :returns: DataFrame with sorted values or None if ddf=None.
+        """
+        keys = None
+        if isinstance(by, str):
+            keys = [by]
+        elif isinstance(by, list):
+            keys = by
+        else:
+            raise ValueError('the value sorted by should either be string or list of string')
+
+        readers = tuple(self._columns[k] for k in keys)
+
+        sorted_index = self._dataset.session.dataset_sort_index(
```

Comment:
we should move dataset_sort_index to ops and have the session API method call through to common code. We may also want to consider moving from sorting each index in turn to sorting with a numpy.lexsort, although we may need to enable high-scale sorting if we do that; let's have a chat about that.

Reply:
I saw that some other methods in dataframe.py also call session API method by self._dataset.session, e.g. create_**_field. Do we also need to replace that as well?